test/parallel/test-cluster-send-deadlock.js

```diff
@@ -30,42 +30,42 @@ const net = require('net');
 
 if (cluster.isMaster) {
   const worker = cluster.fork();
-  worker.on('exit', function(code, signal) {
+  worker.on('exit', (code, signal) => {
     assert.strictEqual(code, 0, `Worker exited with an error code: ${code}`);
     assert(!signal, `Worker exited by a signal: ${signal}`);
     server.close();
   });
 
-  const server = net.createServer(function(socket) {
+  const server = net.createServer((socket) => {
     worker.send('handle', socket);
   });
 
-  server.listen(0, function() {
+  server.listen(0, () => {
     worker.send({ message: 'listen', port: server.address().port });
   });
 } else {
-  process.on('message', function(msg, handle) {
+  process.on('message', (msg, handle) => {
     if (msg.message && msg.message === 'listen') {
       assert(msg.port);
       const client1 = net.connect({
         host: 'localhost',
         port: msg.port
-      }, function() {
+      }, () => {
         const client2 = net.connect({
           host: 'localhost',
           port: msg.port
-        }, function() {
+        }, () => {
           client1.on('close', onclose);
           client2.on('close', onclose);
           client1.end();
           client2.end();
         });
       });
       let waiting = 2;
-      function onclose() {
+      const onclose = () => {
```

Comment:
my guidance was to skip named functions from being converted to arrows, as I wasn't sure about implications. Just stating here so. Did linter say anything specifically on this?

Reply:
I did give it some thought and read around a bit — didn't find anything strong against it, though.  Also, no complaints from the linter.

I inferred from [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) that arrow functions are generally a better fit for non-method functions, as long as they are not going to be used as constructors.

Reply:
@sagirk - sure, thanks for the clarification!